### PR TITLE
Accessibility: Improve Image Slideshow button label

### PR DESCRIPTION
**Pull request by nickcolley**
Previously the label read `control animation`. Without context it's not clear what this means, what does control mean and what animation does it refer to?

The new label will read 'Play "{title}" image slide show' for example 'Play "FT Brexit and Beyond Summit" image slideshow'.

I have chosen the word `slideshow` as I hope this is more descriptive of the animation.

If the animation is running it will say "Pause" instead of "Play".

In the future, I agree with the Digital Accessibility Center (DAC) report that we should reconsider this animation all together, could we remove it and keep a single fixed image?

DAC ID: IDAC_Non-Descriptive_Form_Elements_Issue4

See also: https://sarahmhigley.com/writing/playing-with-state/

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -47,3 +47,33 @@ Property             | Type                | Notes
...
+
+This package does not have a demonstration application, so to test we can link to [`n-magnet` repostiory](https://github.com/Financial-Times/n-magnet) which consumes this.
+
+Complete the installation for `n-magnet` locally then link `n-eventpromo` by running:
+
+```sh
+cd ../n-magnet
+npm link ../n-eventpromo
+make build && make demo
+```
```
Comment (andygout) [inline readme.md:70]: 😍 Love a bit of documentation.

File: src/ImagesContainer.jsx [hunk 2]
```diff
@@ -11,15 +11,16 @@ const withPauseActions = withActions(() => ({
 	}
 }));
 
-const BaseImagesContainer = ({ isPaused, actions: { togglePause }, images, link }) => {
+const BaseImagesContainer = ({ isPaused, actions: { togglePause }, images, link, title }) => {
```
Comment (andygout) [inline src/ImagesContainer.jsx:14]: Is `title` included in the props argument as default or will we need to add it where `BaseImagesContainer` is used?

Comment (nickcolley) [reply src/ImagesContainer.jsx:14]: The parent passes through everything in a `{...props}` splat. I prefer explicit passing but didn't feel appropriate to refactor this.

**Comment by nickcolley** [comment]
@andygout I've bumped this project to use Node.js 12 so the build will succeed (master) is failing. Can you review and see if you're happy with that?

File: package.json [hunk 3]
```diff
@@ -49,7 +49,7 @@
   },
   "homepage": "https://github.com/Financial-Times/n-eventpromo",
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 12.0.0"
```
Comment (andygout) [inline package.json:52]:
```suggestion
    "node": "12.x"
```

To prevent it running on v13, which does not have LTS.

**Pull request merged**